bridgestyle/arcgis/togeostyler.py

```diff
@@ -339,6 +342,8 @@ def _formatPolygonSymbolizer(symbolizer, markerPlacement):
     return symbolizer
 
 def _processOrientedMarkerAtEndOfLine(layer, options):
+    markerPosition = layer["markerPlacement"]["positionArray"][0]
+    flipFirst = layer["markerPlacement"]["flipFirst"]
```

Comment:
Is it possible to not have this attribute, should we put a default value here ?
And is it possible to not having the `markerPlacement` ? If that's the case, the application can crash.
Can you check please ? Thanks.

Reply:
Thanks, good points. If we get here `markerPlacement` do exist, but we can't be sure for `flipFirst` -> I'll add a default value for this one